packages/extension-commerce-bm-seo/src/setup-app.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Third-Party
+import React from 'react'
+import {RouteProps} from 'react-router-dom'
+
+// Platform Imports
+import {
+    ApplicationExtension,
+    SliceInitializer,
+    withApplicationExtensionStore
+} from '@salesforce/pwa-kit-extension-sdk/react'
+import {applyHOCs} from '@salesforce/pwa-kit-extension-sdk/react/utils'
+import {
+    AsyncRouteProps,
+    ComponentMap,
+    GetRoutesParams,
+    BeforeRouteMatchParams
+} from '@salesforce/pwa-kit-extension-sdk/types'
+
+// Local Imports
+import {Config} from './types'
+import {
+    createPlaceholderComponent,
+    getComponentForUrlMapping,
+    withPropsWrapper
+} from './utils/component-utils'
+import {getAppOrigin, getShopperSeoClient} from './utils/shopper-seo-utils'
+
+// Others
+import extensionMeta from '../extension-meta.json'
+
+interface StoreSlice {
+    count: number
+    increment: () => void
+    decrement: () => void
+}
+
+// This is safe to delete if your extension does not use state. If you aren't using this, ensure you remove the
+// `withApplicationExtensionStore` usage below as well.
+const sliceInitializer: SliceInitializer<StoreSlice> = (set) => ({
+    count: 0,
+    increment: () => set((state) => ({count: state.count + 1})),
+    decrement: () => set((state) => ({count: state.count - 1}))
+})
```

Comment:
We can probably toss this is we aren't using it. I can't remember if Jer is using it or not tho.